### ISSUE TITLE
[Feature]: automated documentation of current car architecuture

### ISSUE THREAD

**Issue by asamluka** [una-auxme/paf#552]
### Feature Description

The documentation gets outdated pretty fast. Keeping it up to date manually is a time consuming procedure. Automating this would save time in the long run and ensures, that reader of the documentation always see the newest build.

### Definition of Done

- [ ] simple script that gathers needed information
- [ ] autogeneration of markdown file with current information about subscribers and publishers 

### Testability

Generate a markdown and compare to current code or rqt graph.

**Comment by SirMDA** [comment]
@asamluka Wird der Issue noch weiter verfolgt?

**Comment by asamluka** [comment]
@SirMDA Nein, es hat eine niedrige Priorität und die Kapa wird derzeit anderweitig benötigt. Dennoch fand ich es sinnvoll die Idee irgendwo abzulegen

**Comment by Johannes1098** [comment]
Idea transfer to PAF25
